openmls/src/group/mls_group/test_mls_group.rs

```diff
@@ -1142,16 +1145,60 @@ fn remove_prosposal_by_ref(
 
 // Test that the builder pattern accurately configures the new group.
 #[openmls_test]
-fn group_context_extensions_proposal() {
+fn group_context_extensions_proposal<Provider: OpenMlsProvider + Default>() {
```

Comment:
Doesn't the `openmls_test` macro already give you the `Provider` type to create a new provider?